airqo-platform/AirQo-api
# Update calibrate api

# <Feature/Chore/Hotfix/Topic/Bug Title>

**_WHAT DOES THIS PR DO?_**
Updates  calibrate API 

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
https://airqoteam.atlassian.net/jira/software/projects/DP/boards/44?assignee=5d91eed256bb950dd6489270&selectedIssue=DP-62

**_WHAT IS THE LINK TO THE PR BRANCH_**
https://github.com/airqo-platform/AirQo-api/tree/update-calibrate-api

**_HOW DO I TEST OUT THIS PR?_**
cd AirQo-api/src/calibrate
python -m venv [DESTINATION_FOLDER]
 [DESTINATION_FOLDER]/Scripts/activate
pip install -r requirements.txt

python jobs\regression.py  (one time)
flask run
NB: Make sure you add a GCP service account key under jobs/

SAMPLE REQUEST BODY
{ 
"datetime": "2020-07-15 13:00:00",
 "raw_values":[
          {
              "device_id":"aq_01",
               "sensor1_pm2.5": 44.12 , 
               "sensor1_pm10":54.20,
               "sensor2_pm2.5": 44.12 , 
               "sensor2_pm10":54.20,            
               "temperature":25.3, 
               "humidity":62.0
         }, 
         { 
                "device_id":"aq_02",
                "sensor1_pm2.5": 44.12 , 
               "sensor1_pm10":54.20,
               "sensor2_pm2.5": 44.12 , 
               "sensor2_pm10":54.20,          
                "temperature":26.5, 
                "humidity":57.0 
          }, 
          { 
                  "device_id":"aq_03",
                  "sensor1_pm2.5": 44.12 , 
                  "sensor1_pm10":54.20,
                 "sensor2_pm2.5": 44.12 , 
                 "sensor2_pm10":54.20,  
                  "temperature":27.0, 
                  "humidity":56.0 
            } 
                        ]
 }




**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
[POST] http://127.0.0.1:4001/api/v1/calibrate 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
NA

**_ARE THERE ANY RELATED PRs?_**
NA
